backend/src/main/resources/db/migration/V12__second_dose.sql

```diff
@@ -0,0 +1,15 @@
+-- prepare db for second dose of vaccine
+ALTER TABLE vaccinations
+    ADD COLUMN dose_number INTEGER NOT NULL DEFAULT 1;
+
+ALTER TABLE patients
+    ADD COLUMN vaccination_second_dose_id UUID REFERENCES vaccinations (id) ON DELETE SET NULL;
+
+-- Drops UNIQUE constraint of vaccinations.patient_id.
+-- The constraint name vaccinations_patient_id_key is not specified in any migration however
+-- the standard names in PostgreSQL are {tablename}_{columnname(s)}_{suffix} so this should work.
+ALTER TABLE vaccinations
+    DROP CONSTRAINT vaccinations_patient_id_key;
+
+ALTER TABLE vaccinations
+    ADD UNIQUE (patient_id, dose_number);
```

Comment:
nakonec bych asi dropnul ten default 1 u dose_number. To tam mame jenom pro tu migraci, aby se tam doplnily ty chybejici 1. Ale z podstaty veci 1 defaultni hodnota neni, takze bych to zas dropnul (i to pripadne pomuze, protoze to bude padat, az do DB budeme ukladat davku bez cisla)